src/test/java/net/snowflake/client/jdbc/PreparedStatement2IT.java

```diff
@@ -337,162 +329,162 @@ public void testSnow35923() throws Exception {
   @Test
   @ConditionalIgnoreRule.ConditionalIgnore(condition = RunningOnGithubAction.class)
   public void testBindObjectLiteral() throws Exception {
+    long t1Id = 0;
+    long t2Id = 0;
+
     try (Connection conn = init()) {
-      Statement stmt = conn.createStatement();
-
-      String sqlText = "create or replace table identifier(?) (c1 number)";
-      SnowflakePreparedStatementV1 pStmt =
-          (SnowflakePreparedStatementV1) conn.prepareStatement(sqlText);
-      String t1 = "bindObjectTable1";
-      // Bind the table name
-      pStmt.setString(1, t1);
-      ResultSet result = pStmt.executeQuery();
-
-      // Verify the table has been created and get the table ID
-      stmt.execute("select parse_json(system$dict_id('table', '" + t1 + "')):entityId;");
-      result = stmt.getResultSet();
-
-      long t1Id = 0;
-      if (result.next()) {
-        t1Id = Long.valueOf(result.getString(1));
-      }
+      try (Statement stmt = conn.createStatement()) {
```

Comment:
You could just add this to the try above

Reply:
Fixed.